parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -124,10 +125,20 @@ public Metrics metrics() {
     return ParquetUtil.footerMetrics(writer.getFooter(), model.metrics(), metricsConfig);
   }
 
+  /**
+   * Returns the number of bytes written by this writer.
+   * NOTE: call {@link ParquetWriter#close()})} prior this method to get the exact number of bytes.
```

Comment:
This is going to be wrapped into a single paragraph. Also, it is okay to describe the behavior without `NOTE`.

How about this?

```java
  /**
   * Returns the approximate length of the output file produced by this writer.
   * <p>
   * Prior to calling {@link ParquetWriter#close}, the result is approximate. After calling close, the length is exact.
   */
```

Reply:
sounds good